src/Symfony/Component/Console/Tests/Question/ChoiceQuestionTest.php

```diff
@@ -19,21 +19,21 @@ class ChoiceQuestionTest extends TestCase
     /**
      * @dataProvider selectUseCases
      */
-    public function testSelectUseCases($multiSelect, $answers, $expected, $message)
+    public function testSelectUseCases($multiSelect, $answers, $expected, $message, $default = null)
     {
         $question = new ChoiceQuestion('A question', [
             'First response',
             'Second response',
             'Third response',
             'Fourth response',
-        ]);
+            null,
+        ], $default);
 
         $question->setMultiselect($multiSelect);
 
         foreach ($answers as $answer) {
             $validator = $question->getValidator();
             $actual = $validator($answer);
-
```

Comment:
Can you revert this one? Unrelated changes should be avoided

Reply:
Done.